forward_engineering/dbtProvider.js

```diff
@@ -60,6 +61,18 @@ class DbtProvider {
 
 		return decorateType(type, columnDefinition);
 	}
+
+	/**
+	 * @param {{ jsonSchema: Record<string, unknown> }}
+	 * @returns {CompositeKeyConstraintDto[]}
+	 */
+	getCompositeKeyConstraints({ jsonSchema }) {
+		const { getCompositePrimaryKeys, getCompositeUniqueKeys } = require('./helpers/keyHelper')(this.#appInstance);
```

Comment:
@serhii-filonenko i'm really not convinced by the need for dynamic require here, do we really need it?

Reply:
@bigorn0 Are you suggesting to import a function that returns a helper at the top level and call it inside the method to get the helper?

Reply:
just wondering why it can't just be a module by itself directly imported at the top yes indeed

Reply:
I get that plugins are made that way for now but all of these dynamic require are a bit convoluted tbh and don't make dependency tree obvious....
Probably a question for later :)